language/metrics-output-plugin/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./metrics-output";
 export * from "./utils";
+export { MetricsContent, MetricsReport, MetricItem } from "./types";
```

Comment:
Should we just export everything?

Reply:
I know we discussed it, but we also discussed only exposing the types that were being used. For the sake of simplifying this though, I think exporting everything would be the safest and least disruptive if we need more types later on. 

Will update.